apps/backend/src/database/migrate.ts

```diff
@@ -0,0 +1,38 @@
+import { MigrationExecutor } from 'typeorm';
+import db from './datasource';
+import { Context } from 'aws-lambda';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export const handler = async (event?: unknown, context?: Context) => {
+  console.log('Starting migrations...');
```

Comment:
I think we can mark any unused placeholder args with "_" ie: "_argname" and eslint will not complain? 

(Also cool that this is showing up in the PR checks!)

Reply:
renamed